polkadot-protocol/smart-contracts-basics/overview.md

```diff
@@ -0,0 +1,152 @@
+---
+title: Smart Contracts Basics Overview
+description: Learn how developers can build smart contracts on Polkadot by leveraging either Wasm/ink! or EVM contracts across many parachains.
+---
+
+# An Overview of the Smart Contract Landscape on Polkadot
+
+## Introduction
+
+Polkadot is designed to support an ecosystem of parachains, rather than hosting smart contracts directly. Developers aiming to build smart contract applications on Polkadot rely on parachains within the ecosystem that provide smart contract functionality.
+
+This guide outlines the primary approaches to developing smart contracts in the Polkadot ecosystem:
+
+- **Wasm-based smart contracts** - using [ink!](https://use.ink/){target=\_blank}, a Rust-based embedded domain-specific language (eDSL), enabling developers to leverage Rust’s safety and tooling
+- **EVM-compatible contracts** - which support languages like [Solidity](https://soliditylang.org/){target=\_blank} and [Vyper](https://vyperlang.org/){target=\_blank}, offering compatibility with popular Ethereum tools and wallets
+- **PolkaVM-compatible contracts** - which support Solidity and Rust while maintaining compatibility with Ethereum based tools
```

Comment:
Reorder this to:
- PolkaVM
- EVM-compatible
- Wasm

And do the same in the order of the sections